src/Panel/Panel.jsx

```diff
@@ -0,0 +1,48 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PropTypes} from "react";
+
+import "./Panel.less";
+
+/*
+ * Panel is various types of panel.
+*/
+export default function Panel({children, type, title}) {
+  const {Icons, cssClass} = Panel;
+
+  return (
+    <div className={classnames(`Panel--${type}`, cssClass.CONTAINER)}>
+      <div className={cssClass.HEADER}>
+        <i className={classnames("fa", Icons[type])} />
```

Comment:
Would add an 'alertIcon--[type]' here to style directly rather than have them nested in the LESS.